### PR TITLE
Fetch projects and list them in the projects view

### LINKED ISSUE #53: Display current user projects
Create new page with all projects for current user cf https://docs.gitlab.com/ee/api/projects.html#list-all-projects

### PR DESCRIPTION
**Pull request by franciscoemanuel**
Resolves #53. If there are any adjustments, or suggestions I'd love to hear it from you guys.

### CODE REVIEW

File: src/components/Project.vue [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            <v-icon small>star</v-icon>
+            <span class="caption">{{ star_count }}</span>
+          </span>
+          <span>Stars</span>
+        </v-tooltip>
+
+        <v-tooltip bottom>
+          <span class="pr-3" slot="activator">
+            <v-icon small>call_split</v-icon>
+            <span class="caption">{{ forks_count }}</span>
```
Comment (chamerling) [inline src/components/Project.vue:36]: What about using the same layout as for merge requests icons?

File: src/services/gitlab/client.js [hunk 2]
```diff
@@ -148,4 +148,10 @@ export default class Client {
       .post(`/api/v4/projects/${project_id}/merge_requests/${iid}/todo`)
       .then(result => result.data);
   }
+
+  fetchProjects() {
+    return this.client
+      .get(`/api/v4/projects?membership=true`)
```
Comment (chamerling) [inline src/services/gitlab/client.js]: Maybe default to `starred` projects first and order them by `name`

File: src/views/Projects.vue [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  components: {
+    Project
+  },
+  computed: {
+    ...mapGetters({
+      projects: "getProjects",
+      projectSize: "getProjectsSize"
+    })
+  },
+  methods: {}
```
Comment (chamerling) [inline src/views/Projects.vue]: useless empty

File: src/App.vue [hunk 4]
```diff
@@ -9,128 +9,135 @@
         hide-overlay
         app
       >
-
```
Comment (chamerling) [inline src/App.vue]: When adding a single element, I expect to not change the whole component. Please update this file to reflect only the change to add the projects in the sidebar

File: src/services/gitlab/client.js (hunk 2 above)
Comment (franciscoemanuel) [reply src/services/gitlab/client.js]: Do you mean to only list starred projects? or to first order by starred projects and then by name?

Comment (chamerling) [reply src/services/gitlab/client.js]: Display only starred projects and order them by name. Parameters are available in the API

**Comment by franciscoemanuel** [comment]
Any more suggestions?

**Comment by chamerling** [comment]
> Any more suggestions?

Looks good @franciscoemanuel !
Resolve conflicts, force push and then we can merge it

**Comment by franciscoemanuel** [comment]
Done 👍

**Pull request merged**